gradle.properties

```diff
@@ -33,7 +33,7 @@ systemProp.org.gradle.internal.http.socketTimeout=120000
 
 GROUP=com.google.accompanist
 # !! No longer need to update this manually when using a Compose SNAPSHOT
-VERSION_NAME=0.24.3-SNAPSHOT
+VERSION_NAME=0.24.3-alpha
```

Comment:
Why does this need to be changed?

Reply:
This is the version number that gets sent to maven, it's always changed in these PRs and then quickly changed back to snapshot after release. That comment above it is for when dealing with making new snapshot builds.